rules/static.go

```diff
@@ -645,13 +645,6 @@ func (r *DefaultRuleRenderer) filterOutputChain(ipVersion uint8) *Chain {
 		},
 	)
 
-	// Jump to chain for blocking service CIDR loops.
-	rules = append(rules,
-		Rule{
-			Action: JumpAction{Target: ChainCIDRBlock},
-		},
-	)
-
```

Comment:
Does that mean we disable service loop prevention for packet generated by local host?

Reply:
Yes, exactly; but if there is a loop and the packet returns to us; _then_ we'll drop it in the forward chain before we send it back out again.

Reply:
Nice reasoning and solution!